gsl_simulation/views/simulation_projet_annotations_views.py

```diff
@@ -55,3 +60,33 @@ def post(self, request, *args, **kwargs):
             context = self.get_context_data(**kwargs)
             context["projet_note_form"] = form
             return render(request, self.template_name, context)
+
+    def delete(self, request, *args, **kwargs):
+        simulation_projet = get_object_or_404(
+            SimulationProjet, id=self.kwargs.get("pk")
+        )
+        note_id = request.POST.get("note_id")
+        if note_id:
+            note = ProjetNote.objects.get(pk=note_id)
```

Comment:
Ici on a un oracle ! Imaginons qu'un utilisateur mal intentionné essaie de savoir combien de notes existent en base. Il pourrait spammer le endpoint de suppression jusqu'à obtenir un 404 à la place d'un 403.

Du coup ma technique en général c'est assez rustique : 

```suggestion
            note = get_object_or_404(ProjetNote, pk=note_id, created_by=request.user)
```

On a du 404 pour tous les cas où ça marche pas, quelle que soit la raison pour ça.